tests/console/yast2_rmt.pm

```diff
@@ -0,0 +1,86 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test for the yast2-rmt module
+# Maintainer: Jonathan Rivrain <JRivrain@suse.com>
+
+use base "console_yasttest";
+use strict;
+use utils;
+use testapi;
+
+
+sub password_twice {
```

Comment:
there is function `type_password` , maybe we need to use it instead of just `type_string` ?